CONTRIBUTING.md

```diff
@@ -54,26 +54,22 @@ code changes and the commit message should start with `Docs:`.
 
 ### Run the docs locally
 
-Make docs changes and test them by using hugo to run the kpt site locally. Hugo
-continuously builds your website as you make changes.
-
-- install hugo
-- `make servedocs`
+Make docs changes and test them by running the site in a docker container
```

Comment:
why docker?

Reply:
We already have workflows that utilize `docker` and it allows for the most similar dev experience to the live site without relying directly on `firebase`.